clearpath_sensors/launch/velodyne_lidar.launch.py

```diff
@@ -56,7 +56,7 @@ def generate_launch_description():
         namespace=namespace,
         parameters=[parameters],
         remappings=[
-          ('/diagnostics', 'diagnostics')
+          ('/diagnostics', 'diagnostics'), ('/tf', 'tf'), ('/tf_static', 'tf_static')
```

Comment:
The new remaps should not be here as the velodyne driver node does not use those topics